packages/api-clients/open-api/bffApi.yml

```
@@ -11955,144 +12025,781 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  "/status":
+  /delegations:
+    parameters:
+    - $ref: "#/components/parameters/CorrelationIdHeader"
     get:
-      security: []
       tags:
-        - health
-      summary: Health status endpoint
-      description: Return ok
-      operationId: getStatus
+        - delegations
+      summary: List producer delegations
+      description: List producer delegations
+      operationId: getProducerDelegations
```

```suggestion
      summary: List delegations
      description: List delegations
      operationId: getDelegations
```
Since this route takes a `DelegationKind` as input, shouldn't it be used for both kinds?